android-components/components/browser/menu/src/test/java/mozilla/components/browser/menu/BrowserMenuPositioningTest.kt

```diff
@@ -41,6 +43,56 @@ class BrowserMenuPositioningTest {
         Assert.assertEquals(expected, result)
     }
 
+    @Test
+    fun `GIVEN inferMenuPositioningData WHEN availableHeightToBottom is bigger than availableHeightToTop THEN it returns a new MenuPositioningData populated with all data needed to show a PopupWindow that fits down`() {
+        val view: ViewGroup = mock()
+        Mockito.doReturn(70).`when`(view).measuredHeight
+        val anchor = View(testContext)
+        anchor.layoutParams = ViewGroup.LayoutParams(20, 40)
+
+        setScreenHeight(50)
+
+        val result = inferMenuPositioningData(view, anchor, MenuPositioningData())
+
+        val expected = MenuPositioningData(
+            BrowserMenuPlacement.AnchoredToTop.Dropdown(anchor), // orientation DOWN and fitsDown
+            askedOrientation = BrowserMenu.Orientation.DOWN, // default
+            fitsUp = false, // availableHeightToTop(0) is smaller than containerHeight(70) and smaller than availableHeightToBottom(50)
+            fitsDown = true, // availableHeightToBottom(50) is smaller than containerHeight(70) and bigger than availableHeightToTop(0)
+            availableHeightToTop = 0,
+            availableHeightToBottom = 50, // mocked by us above
+            containerViewHeight = 70, // mocked by us above
+        )
+        Assert.assertEquals(expected, result)
+    }
+
+    @Test
+    fun `GIVEN inferMenuPositioningData WHEN availableHeightToTop is bigger than availableHeightToBottom THEN it returns a new MenuPositioningData populated with all data needed to show a PopupWindow that fits down`() {
```

Comment:
should this be: 
```suggestion
    fun `GIVEN inferMenuPositioningData WHEN availableHeightToTop is bigger than availableHeightToBottom THEN it returns a new MenuPositioningData populated with all data needed to show a PopupWindow that fits up`() {
```
?

Reply:
@t-p-white Yes. Updated.